### ISSUE TITLE
Bug in ACGAN implementation

### ISSUE THREAD

**Issue by hmrishavbandy** [eriklindernoren/PyTorch-GAN#141]
https://github.com/eriklindernoren/PyTorch-GAN/blob/a163b82beff3d01688d8315a3fd39080400e7c01/implementations/acgan/acgan.py#L100

Later on we use Cross Entropy with this loss--however, we need raw logits for cross entropy loss...Maybe change it to NLL loss?
